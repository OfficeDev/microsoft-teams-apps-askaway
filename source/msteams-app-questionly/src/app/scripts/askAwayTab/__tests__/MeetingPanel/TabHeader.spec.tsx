/**
 * @jest-environment jsdom
 */

import * as React from 'react';
import { shallow, configure } from 'enzyme';
import enzymeToJson from 'enzyme-to-json';
import { Menu } from '@fluentui/react-northstar';
import Adapter from 'enzyme-adapter-react-16';
import TabHeader from '../../MeetingPanel/TabHeader';

configure({ adapter: new Adapter() });

describe('AskAwayTabRemove Component', () => {
    const t = jest.fn();
    const tabActiveIndex = 1;
    const onSelectActiveTab = jest.fn();
    it('should match the snapshot', () => {
        const wrapper = shallow(<TabHeader t={t} tabActiveIndex={tabActiveIndex} onSelectActiveTab={onSelectActiveTab} />);
        expect(enzymeToJson(wrapper)).toMatchSnapshot();
    });

    it('should render the tab', () => {
        const component = shallow(<TabHeader t={t} tabActiveIndex={tabActiveIndex} onSelectActiveTab={onSelectActiveTab} />);

        expect(component.find(Menu)).toBeTruthy();
    });
});
