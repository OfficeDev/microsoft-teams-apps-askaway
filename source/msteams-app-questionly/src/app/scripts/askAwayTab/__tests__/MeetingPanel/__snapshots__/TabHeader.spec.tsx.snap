// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AskAwayTabRemove Component should match the snapshot 1`] = `
<Fragment>
  <Menu
    accessibility={[Function]}
    aria-label="Today's events"
    as="ul"
    className="menu-bar"
    defaultActiveIndex={1}
    items={
      Array [
        Object {
          "content": undefined,
          "key": "unansweredQuestions",
          "onClick": [Function],
        },
        Object {
          "content": undefined,
          "key": "answeredQuestions",
          "onClick": [Function],
        },
      ]
    }
    primary={true}
    underlined={true}
  />
</Fragment>
`;
